src/link/colorScheme.js

```diff
@@ -0,0 +1,24 @@
+const attNameWPDarkModePlugin = 'data-wp-dark-mode-active';
+
+const isWPDarkModePluginActive = () => {
+	return document.documentElement.hasAttribute( attNameWPDarkModePlugin );
+};
+
+export const getColorScheme = () => {
+	if ( isWPDarkModePluginActive() ) {
```

Comment:
what if user toggle the plugin to light mode but device setting is dark mode, should it set to light mode? currently detect shows dark mode.

Reply:
When the system setting is set to dark and the user toggles the plugin to light, the expected behavior is that the preview renders light. This is achieved via the `callback('light')` in line 20 below, I am seeing the expected behavior in my set up. Could you try again to confirm?

Reply:
that's true, got it thanks.